src/elm/Styles/Colours.elm

```diff
@@ -0,0 +1,78 @@
+module Styles.Colours exposing (getPastel)
+
+import Styles.Classes exposing (tachs)
+import Color exposing (..)
+import Array exposing (..)
+import Maybe exposing (withDefault)
+
+
+type alias Swatches =
+    Array RgbString
+
```

Comment:
Although, I imagine you'd want unique values if you're making colour swatches, and in that case a structure like `Set` would *enforce* that they were unique